### DISCUSSION TITLE
Zimbra Daffodil (v10.1.2) supports EL9 now

### DISCUSSION THREAD

**Discussion by dinkonin** [ianw1974/zimbra-build-scripts#97] [General]
Are there plans to support this in the script and to relase a prebuild image for el9?

**Comment by ianw1974** [comment]
HI yeah, there will be.  TBH I hadn't realised that Zimbra started supporting it.  I'll get it done shortly.

**Reply by dinkonin** [reply]
I'm a little bit impatient so I've built it, FYI other than adding `"${DISTRIB_ID}" == "RockyLinux" ` and uncommenting the stuff in el9_pkg_install(), (and removing java11 from the VM that I use to build ) there were no problems.

Thank your for your awesome work!

**Reply by ianw1974** [reply]
My pleasure.  Thx for the info, I'm gonna check the dependencies anyway as I see a ```maven-openjdk8``` package that will most likely work better than the standard ```maven``` one, so will see how it works out.

**Comment by ianw1974** [comment]
All done, script should now build for RHEL9 and Rocky 9 at least.  Assuming the info reported by lsb_release for Almalinux hasn't changed, then that should be fine too.  If I get time, I'll install it to check/verify, but otherwise we have it covered :)
